src/openrct2/localisation/Formatting.cpp

```diff
@@ -212,6 +245,105 @@ namespace OpenRCT2
         }
     }
 
+    template<size_t TDecimalPlace, bool TDigitSep, typename T> void FormatCurrency(std::stringstream& ss, T rawValue)
+    {
+        auto currencyDesc = &CurrencyDescriptors[gConfigGeneral.currency_format];
+        auto value = static_cast<int64_t>(rawValue) * currencyDesc->rate;
+
+        // Negative sign
+        if (value < 0)
+        {
+            ss << '-';
+            value = -value;
```

Comment:
`std::abs`

Reply:
Since we already know that it is negative, `abs` is unnecessary.